### ISSUE TITLE
Scrollbars

### ISSUE THREAD

**Issue by slowbackspace** [trezor/trezor-suite#2226]
The problem is that native scrollbars are ugly on most platform except macOS. We can restyle them via CSS for chrome-based browsers that would probably cover most of our userbase (and ofc electron apps too). 

The other option, that should also cover firefox users, would be to use a library such as https://github.com/Grsmto/simplebar, we tried once and failed. As I recall there was some weird issue in accounts menu. It is certainly worth another shot though!

Current state is that in some critical places such as accounts menu, app navigation and modals we have implemented the first option (and it works quite nicely).

Basically someone (I guess that's me) needs to sit down and figure out if the first option (css-way) will be enough (confirmed: it is) and if it should be use globally, or if there is some catch and it needs to be used on component basis. (ohh, the catch is native scrollbars on mac have auto hide functionality, restyled css scrollbars don't allow that, but I guess it's fine).

update: ohh, since firefox 64 there is a way to customize scrollbars thanks to the `scrollbar-color` prop. REJOICE.

**Comment by bosomt** [comment]
**QA OK**

Firefox 82.0 (64-bit) + Nixos
Suide web dev maser commit da17a85
Device mdoel T fw 2.3.4 commit 50854b921
